lib/ts/querier.ts

```diff
@@ -103,6 +103,16 @@ export class Querier {
         return new Querier(Querier.hosts, rIdToCore);
     }
 
+    static getNewCustomInstanceOrThrowError(
+        hosts: { domain: NormalisedURLDomain; basePath: NormalisedURLPath }[],
+        rIdToCore?: string
+    ): Querier {
+        if (!Querier.initCalled) {
+            throw Error("Please call the supertokens.init function before using SuperTokens");
+        }
+        return new Querier(hosts, rIdToCore);
+    }
```

Comment:
this function should not be needed.

Reply:
this way I don't need to implement all helpers around doFetch, like retries, etc.

Reply:
The purpose of the querier is to only query the core, nothing else. So we should not misuse it. You are welcome to create a separate module to query the new microservice, but we should not use querier for it

Reply:
ok